src/scribe_data/cli/total.py

```diff
@@ -367,11 +367,15 @@ def total_wrapper(
     """
     # Handle --all flag
     if all_bool and wikidata_dump:
-        language = "all"
+        if data_type is None:
+            data_type = "all"
+        if language is None:
+            language = "all"
```

Comment:
nit - a quick suggestion:

The in-source docs here and elsewhere in the repo mention the following: 
```
    all_bool : boolean
        Whether all languages and data types should be listed.
```
This seems a tad misleading since setting a specific `language` and/or `data_type` will process only for all for that `language` and/or `data_type` specified. Perhaps something with the below slight adjustment might make sense?
```
    all_bool : boolean
        Whether all languages and data types should be listed, unless otherwise specified.
```